openprescribing/frontend/models.py

```diff
@@ -448,6 +458,112 @@ def current_version(self):
                 next_version = version.replaced_by
         return version
 
+    @cached_property
+    def dmd_objs(self):
+        '''Return highest level dm+d objects with this presentation's BNF code.
```

Comment:
I think this could do with a bit more explanation, or linking somewhere with one.

e.g. (but this is hard to write well!)  "because BNF codes are most often mapped by NHS BSA to a VMP plus several of its family members, we are usually unable to distinguish between prescribing for at a VMP level or a VMPP level in the data. Therefore we return the highest level dm+d object, which is actually the most accurate level about which we can make assertions"